src/main/java/com/google/impactdashboard/data/IAMBindingDatabaseEntry.java

```diff
@@ -12,6 +14,17 @@
   public abstract long getTimestamp();
   public abstract int getBindingsNumber();
 
+  /**
+   * A comparator for sorting entries by their timestamp in ascending order.
+   */
+  public static final Comparator<IAMBindingDatabaseEntry> ORDER_BY_TIMESTAMP = 
+      new Comparator<IAMBindingDatabaseEntry>() {
+    @Override
+    public int compare(IAMBindingDatabaseEntry a, IAMBindingDatabaseEntry b) {
+      return Long.compare(a.getTimestamp(), b.getTimestamp());
+    } 
+  };
```

Comment:
can this just be a lambda? 
(a,b) -> return Long.compare(a.getTimestamp(),b.getTimestamp()